Sources/KSCrashRecording/KSCrash.m

```diff
@@ -245,6 +245,23 @@ - (void) setCatchZombies:(BOOL)catchZombies
     }
 }
 
+- (BOOL)monitorMemoryTerminations
+{
+    return (self.monitoring & KSCrashMonitorTypeMemoryTermination) != 0;
+}
+
+- (void)setMonitorMemoryTerminations:(BOOL)monitorMemoryTerminations
+{
+    if (monitorMemoryTerminations) 
+    {
+        self.monitoring |= KSCrashMonitorTypeMemoryTermination;
+    }
+    else
+    {
+        self.monitoring &= (KSCrashMonitorType)~KSCrashMonitorTypeMemoryTermination;
+    }
+}
```

Comment:
Thanks for the flag, yet it's slightly different from what I meant.

My idea was to keep memory monitoring being enabled/disabled with monitor type and `KSCrashMonitorTypeMemoryTermination`. But have an additional parameter that would disable OOM's reporting (while keeping memory stats in report for other types of crashes).

But, you know, after a while of thinking about it, I'm okay to keep both features coupled for now. In any case, feel free to follow-up if you find this idea useful and let's see if anyone comes with an issue or a PR later.

Reply:
Ah, gotcha, I think I can make that work in a simple way.